db/schema.rb

```diff
@@ -111,6 +111,16 @@
     t.index ["user_id"], name: "ix_comments_on_user_id"
   end
 
+  create_table "committee_decisions", force: :cascade do |t|
+    t.bigint "planning_application_id"
+    t.boolean "recommend", default: false, null: false
```

Comment:
Just one very tiny comment that perhaps `recommended` would be better as it reads more like an attribute but `recommend` is more readable, so probably leave it as is.

Reply:
Haha yes I can argue it either way in my head